platform-tests/src/acceptance/rds_broker_test.go

```diff
@@ -124,6 +127,72 @@ var _ = Describe("RDS broker", func() {
 			Expect(resp.StatusCode).To(Equal(200), "Got %d response testing multi-user permissions. Response body:\n%s\n", resp.StatusCode, string(body))
 		})
 	})
+
+	FContext("creating a database instance with custom parameters", func() {
+
+		const (
+			DB_CREATE_TIMEOUT = 30 * time.Minute
+			testPlanName      = "M-dedicated-9.5"
+			region            = "eu-west-1"
+			expectedTagKey    = "SkipFinalSnapshot"
+			expectedTagValue  = "true"
+		)
+
+		var (
+			dbInstanceName  string
+			rdsInstanceName string
+		)
+
+		BeforeEach(func() {
+			dbInstanceName = generator.PrefixedRandomName("test-db-")
+			Expect(cf.Cf("create-service", serviceName, testPlanName, dbInstanceName, "-c", `{"skip_final_snapshot": true}`).Wait(DEFAULT_TIMEOUT)).To(Exit(0))
+
+			fmt.Fprint(GinkgoWriter, "Polling for RDS creation to complete")
+			Eventually(func() *Buffer {
+				fmt.Fprint(GinkgoWriter, ".")
+				command := quietCf("cf", "service", dbInstanceName).Wait(DEFAULT_TIMEOUT)
+				Expect(command).To(Exit(0))
+				return command.Out
+			}, DB_CREATE_TIMEOUT, 15*time.Second).Should(Say("create succeeded"))
+			fmt.Fprint(GinkgoWriter, "done\n")
+
+			serviceOutput := cf.Cf("service", dbInstanceName).Wait(DEFAULT_TIMEOUT)
+			Expect(serviceOutput).To(Exit(0))
+			rxp, _ := regexp.Compile("rdsbroker-([a-z0-9-]+)")
+			rdsInstanceName = string(rxp.Find(serviceOutput.Out.Contents()))
+			fmt.Fprintf(GinkgoWriter, "Created RDS instance: %s\n", rdsInstanceName)
+		})
+
+		AfterEach(func() {
+			Expect(cf.Cf("delete-service", dbInstanceName, "-f").Wait(DEFAULT_TIMEOUT)).To(Exit(0))
+			// Poll until destruction is complete, otherwise the org cleanup fails.
+			fmt.Fprint(GinkgoWriter, "Polling for RDS destruction to complete")
+			Eventually(func() *Buffer {
+				fmt.Fprint(GinkgoWriter, ".")
+				command := quietCf("cf", "services").Wait(DEFAULT_TIMEOUT)
+				Expect(command).To(Exit(0))
+				return command.Out
+			}, DB_CREATE_TIMEOUT, 15*time.Second).ShouldNot(Say(dbInstanceName))
+			fmt.Fprint(GinkgoWriter, "done\n")
+
+			dbInstance, err := rdshelper.NewRDSDBInstance(region)
+			Expect(err).NotTo(HaveOccurred())
+			snapshots, err := dbInstance.GetDBFinalSnapshots(rdsInstanceName)
+			Expect(err).To(HaveOccurred())
+			Expect(err.Error()).Should(ContainSubstring("DBSnapshotNotFound"))
+			fmt.Fprint(GinkgoWriter, "Final snapshots for %s:\n", rdsInstanceName)
+			fmt.Fprint(GinkgoWriter, snapshots)
```

Comment:
When it comes to error - we check if snapshot was created. If it doesn't exist you will get 404 with error string. If the returned error is not 404 or the string is different than the test should fail.